cypress/e2e/test.cy.js

```diff
@@ -0,0 +1,49 @@
+//User loads the page//
+describe('On Page Load', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://api.thedogapi.com/v1/images/search?limit=100&api_key=live_00isfy9kzQCyFWWBludIQFj4g1pDwEoM87PH2PTVx8njhE7q1oEBDzg5lOhHq0QZ', {
+      statusCode: 200,
+      fixture: 'apiResponse.json', // This fixture should represent the initial state
+    }).as('apiRequest');
+    cy.visit('localhost:3000');
+    cy.wait('@apiRequest');
+  });
```

Comment:
@butlertree Great job in the intercept here! I saw it properly stubbed in the cypress run, as expected. ⭐